server_admin/topics/authentication/flows.adoc

```diff
@@ -2,197 +2,228 @@
+. Click *Add execution*.
+. Select *Password Form* from the drop-down list. 
+. Click *Save*.
+. Click *Required* for the *Password Form* authentication type to set its requirement to required.
+. Click *Actions* for the *Password with OTP* sub-flow.
+. Click *Add execution*.
+. Select *OTP Form* from the drop-down list. 
+. Click *Save*.
+. Click *Required* for the *OTP Form* authentication type to set its requirement to required.
+
+Finally, we must change the bindings.
+
+. Click the *Bindings* tab.
+. Click the *Browser Flow* drop-down list.
+. Select *Browser Password-less* from the drop-down list. 
+. Click *Save*.
 
-* Using the `Actions` menu on the right-hand side of the "Forms" subflow, select "Add execution". Using the drop-down select
-  "Username Form". After pressing "Save", set its Requirement to _Required_.
+.A password-less browser login
+image:images/Passwordless-browser-login.png[]
 
-The Username form is similar to "Browser" flow's Username Password Form, but only asks for a username, allowing a user to perform a password-less login.
-However, note that this inevitably allows a user enumeration attack on your {project_name} server. This is an unavoidable security risk for the convenience,
-so the flow should make sure that an attacker cannot just have to guess a password to be able to enter.
+After entering the username, the flow works as follows:
 
-* Using the `Actions` menu on the right-hand side of the "Forms" subflow, select "Add flow". Choose an representative Alias, e.g. "Authentication".
-  After pressing "Save", set its Requirement to _Required_.
-* Using the `Actions` menu on the right-hand side of the "Authentication" subflow, select "Add execution". Using the drop-down select
-  "Webauthn Passwordless Authenticator". After pressing "Save", set its Requirement to _Alternative_.
-* Using the `Actions` menu on the right-hand side of the "Authentication" subflow, select "Add flow". Choose an representative Alias, e.g. "Password with OTP".
-  After pressing "Save", set its Requirement to _Alternative_.
-* Using the `Actions` menu on the right-hand side of the "Password with OTP" subflow, select "Add execution". Using the drop-down select
-  "Password Form". After pressing "Save", set its Requirement to Required.
-* Using the `Actions` menu on the right-hand side of the "Password with OTP" subflow, select "Add execution". Using the drop-down select
-  "OTP Form". After pressing "Save", set its Requirement to Required.
-* In the "Bindings" menu, change the browser flow from "Browser" to "Browser Password-less"
+If the user has WebAuthn passwordless credentials recorded, they can use them to login directly. This is the password-less login. The user can also select *Password with OTP* because the `WebAuthn Passwordless` execution and the `Password with OTP` flow are set to *Alternative*. If they are set to *Required*, the user has to enter WebAuthn, password, and OTP.
```

Comment:
If users has WebAuthn passwordless credentials recorded, they can use these credentials to log in directly.